operator-common/src/main/java/io/strimzi/operator/common/Annotations.java

```diff
@@ -18,6 +18,12 @@
 
     public static final String STRIMZI_DOMAIN = "strimzi.io/";
     public static final String STRIMZI_LOGGING_ANNOTATION = STRIMZI_DOMAIN + "logging";
+    /**
+     * Annotations for rolling a cluster whenever the logging (or it's part) has changed
```

Comment:
```suggestion
     * Annotations for rolling a cluster whenever the logging (or it's part) has changed.
       By changing the annotation we force a restart since the pod will be out of date compared to the statefulset.
```